libs/protocol-ui/src/components/molecules/GovrnCta/GovrnCta.tsx

```diff
@@ -0,0 +1,60 @@
+import { Heading, Divider, Flex, Text } from '@chakra-ui/react';
+
+export interface GovrnCtaProps {
+  heading: string;
+  emoji?: string;
+  copy?: string;
+  children?: React.ReactNode;
+}
+
+const GovrnCta = ({ heading, emoji, copy, children }: GovrnCtaProps) => (
+  <Flex
+    direction="column"
+    alignItems="center"
+    justifyContent="center"
+    background="white"
+    boxShadow="sm"
+    borderRadius={{ base: 'none', md: 'lg' }}
+  >
+    <Flex direction="row" alignItems="baseline" fontSize="3xl" paddingY={8}>
+      {emoji ? (
+        <Text
+          as="span"
+          role="img"
+          aria-labelledby="eye emoji for the CTA"
+          fontSize="inherit"
+        >
+          {emoji}
+        </Text>
+      ) : null}
```

Comment:
Why not use &&? It is more concise

Reply:
Mainly to avoid potential issues if children evaluates to 0 such as if it's coming in as an empty array. It'll render as a "0"

Reply:
emoji won't have that problem right? And even for children it is probably preferable to move the turnary out of the JSX